src/tests/ftest/vmd/fault_reintegration.py

```diff
@@ -0,0 +1,259 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from multiprocessing import Queue
+import threading
+
+from avocado.core.exceptions import TestFail
+from dmg_utils import get_storage_query_device_info, get_dmg_response
+from exception_utils import CommandFailure
+from ior_utils import run_ior, thread_run_ior
+from job_manager_utils import get_job_manager
+from led import VmdLedStatus
+from nvme_utils import set_device_faulty
+
+
+class NvmeFaultReintegrate(VmdLedStatus):
+    """
+    Test Class Description: This test runs
+    NVME fault and reintegration test cases.
+
+    :avocado: recursive
+    """
+    # pylint: disable=too-many-ancestors
+    def setUp(self):
+        """Set up for test case."""
+        super().setUp()
+        self.daos_command = self.get_daos_command()
```

Comment:
It doesn't look like you're using much else from VmdLedStatus, if so we could remove the inheritance by adding this:
```suggestion
        self.daos_command = self.get_daos_command()
        self.dmg = self.get_dmg_command()
        self.dmg.hostlist = self.hostlist_servers[0]
```

Reply:
done.